packages/react-ui-ag/src/Filters/RadioGroupFilterCard.js

```diff
@@ -43,7 +44,17 @@ export default class RadioGroupFilterCard extends Component {
   }
 
   get fields() {
-    return this.props.fields || []
+    const { fields = [] } = this.props
+    return fields.map(field => {
+      const obj = field
+
+      if (typeof field.label === 'string') {
+        obj.label = { text: field.label, 'data-tid': `radiogroup-radiobutton-${field.label}` }
+      } else {
+        obj.label = Object.assign({ 'data-tid': `radiogroup-radiobutton-${field.label.text}` }, field.label)
```

Comment:
You still cannot do this.  What if `label` is empty in the field or it's not an object?